prow/ANNOUNCEMENTS.md

```diff
@@ -183,6 +183,7 @@ state and no claims of backwards compatibility are made for any external API.
 Note: versions specified in these announcements may not include bug fixes made
 in more recent versions so it is recommended that the most recent versions are
 used when updating deployments.
+ - *Feb 21nd, 2022* `protected: false` disables branch protection. Not to manage branch protection by branchprotector is expressed by `unmanaged: true`.
```

Comment:
adjust the date accordingly after this PR is ready to merge.

Reply:
we could link to the "`protect: false` has ambiguous semantics" issue I reopened yesterday to provide more context

Reply:
Updated.